mspm0-data-gen/src/perimap.rs

```diff
@@ -9,6 +9,15 @@ pub static PERIMAP: RegexMap<&str> = RegexMap::new(&[
     (".*:cpuss", "v1"),
     (".*:iomux", "v1"),
     (".*:tim", "v1"),
+    ("mspm0g.51x:adc", "g151x_g351x"),
+    ("mspm0g..0x:adc", "g110x_g150x_g310x_g350x_l122x_l222x"),
+    ("mspm0g..1x:adc", "g110x_g150x_g310x_g350x_l122x_l222x"),
+    ("mspm0l.22x:adc", "g110x_g150x_g310x_g350x_l122x_l222x"),
+    ("mspm0l1..x:adc", "l110x_l130x_l134x_c110x_03fx"),
+    ("mspm0c110x:adc", "l110x_l130x_l134x_c110x_03fx"),
+    ("msps003fx:adc", "l110x_l130x_l134x_c110x_03fx"),
+    ("mspm0l111x:adc", "l111x"),
+    ("mspm0h321x:adc", "h321x"),
```

Comment:
Just a note, haven't brought up H321x yet, largely because I don't have a board for it.